server/src/main/java/org/candlepin/model/Owner.java

```diff
@@ -493,7 +493,13 @@ public void setContentAccessMode(String contentAccessMode) {
      * @return String the value
      */
     public String getContentAccessModeList() {
-        return this.contentAccessModeList;
+        String[] modeList = contentAccessModeList == null ? new String[0] : contentAccessModeList.split(",");
+        String ammend = "";
+        if (!ArrayUtils.contains(modeList, ContentAccessCertServiceAdapter.DEFAULT_CONTENT_ACCESS_MODE)) {
```

Comment:
A distinction without a difference. If setting it to empty =' entitlement', then 'entitlement is on the list.  Otherwise it feels like it is too confusing.